docs/chapter4-payload-handoff-format.rst

```diff
@@ -797,6 +797,11 @@ payload (in the case that only a single display output is supported by payload).
    format            R           string             * `a8b8g8r8` - 32-bit pixels, d[31:24]=a, d[23:16]=b, d[15:8]=g, d[7:0]=r
                                                     * `a8r8g8b8` - 32-bit pixels, d[31:24]=a, d[23:16]=r, d[15:8]=g, d[7:0]=b
                                                     * `a16b16g16r16` - 64-bit pixels, d[63:48]=a, d[47:32]=r, d[31:16]=g, d[15:0]=b
+   redmask           R           u32                mask to red color intensity.
+   greenmask         R           u32                mask to green color intensity.
+   bluemask          R           u32                mask to blue color intensity.
+   reservedmask      R           u32                mask to reserved color intensity.
+   pixelsperscanline R           u32                number of pixel elements per video memory line.
```

Comment:
Is this the same as width, or perhaps stride?

Reply:
This is definition of pixelsperscanline from UEFI spec chapter 12.9.2 EFI_GRAPHICS_OUTPUT_PROTOCOL:
PixelsPerScanLine Defines the number of pixel elements per video memory line. For performance reasons, or due to hardware restrictions, scan lines may be padded to an amount of memory alignment. These padding pixel elements are outside the area covered by HorizontalResolution and are not visible. For direct frame buffer access, this number is used as a span between starts of pixel lines in video memory. Based on the size of an individual pixel element and PixelsPerScanline , the offset in video memory from pixel element (x, y) to pixel element (x, y+1) has to be calculated as “sizeof( PixelElement ) * PixelsPerScanLine”, not “sizeof( PixelElement ) * HorizontalResolution”, though in many cases those values can coincide. This value can depend on video hardware and mode resolution. GOP implementation is responsible for providing accurate value for this field.